example/package.json

```diff
@@ -20,7 +20,7 @@
     "react": "^17.0.2",
     "react-native": "^0.67.1",
     "react-native-bouncy-checkbox": "^3.0.2",
-    "react-native-phyllo-connect": "^react-native-phyllo-connect-0.3.0",
+    "react-native-phyllo-connect": "file:../react-native-phyllo-connect-0.3.2.tgz",
```

Comment:
@erpankajpatel , should't we remove this ? Or will this be removed after testing?

Reply:
@chiragphyllo it will be remove after testing and final PR will be from release_v0.3* to main , at time it will be removed.